libs/dh/charges/feature-create-prices/src/lib/dh-charges-create-prices.component.html

```diff
@@ -23,108 +23,117 @@ <h3>{{ t("createPrices.title") }}</h3>
-            <watt-error *ngIf="charge.controls['description'].hasError('maxlength')">
+            </watt-field-error>
+            <watt-field-error *ngIf="charge.controls['description'].hasError('maxlength')">
               {{ "shared.fieldRequiredErrorText" | transloco }}
-            </watt-error>
-          </watt-form-field>
-
+            </watt-field-error>
+          </watt-textarea-field>
           <!-- Resolution -->
-          <watt-form-field class="resolution">
-            <watt-label>{{ t("resolution") }} *</watt-label>
-            <watt-dropdown
-              formControlName="resolution"
-              [options]="resolutionOptions"
-              name="validityOption"
-              [showResetOption]="false"
-              [placeholder]="t('createPrices.selectResolution')"
-            >
-            </watt-dropdown>
-            <watt-error *ngIf="charge.controls['resolution'].hasError('required')">
+
+          <watt-dropdown
+            class="resolution"
+            label="{{ t('resolution') }} *"
+            formControlName="resolution"
+            [options]="resolutionOptions"
+            name="validityOption"
+            [showResetOption]="false"
+            [placeholder]="t('createPrices.selectResolution')"
+          >
+            <watt-field-error *ngIf="charge.controls['resolution'].hasError('required')">
               {{ "shared.fieldRequiredErrorText" | transloco }}
-            </watt-error>
-          </watt-form-field>
+            </watt-field-error>
+          </watt-dropdown>
 
           <!-- Effective Date -->
           <div class="effective-date">
-            <watt-form-field>
-              <watt-label>{{ t("createPrices.validFromDateTime") }} *</watt-label>
-              <watt-datepicker formControlName="effectiveDate"></watt-datepicker>
-            </watt-form-field>
-            <watt-error
-              *ngIf="charge.controls['effectiveDate'].hasError('required') && hasSubmitted"
+            <watt-datepicker
+              label="{{ t('createPrices.validFromDateTime') }} *"
+              formControlName="effectiveDate"
             >
-              {{ "shared.fieldRequiredErrorText" | transloco }}
-            </watt-error>
+              <watt-field-error
+                *ngIf="charge.controls['effectiveDate'].hasError('required') && hasSubmitted"
+              >
+                {{ "shared.fieldRequiredErrorText" | transloco }}
+              </watt-field-error>
 
-            <watt-error
-              *ngIf="charge.controls['effectiveDate'].hasError('maxDate') && hasSubmitted"
-            >
-              {{ t("createPrices.maxDateValidationErrorText") }}
-            </watt-error>
+              <watt-field-error
+                *ngIf="charge.controls['effectiveDate'].hasError('maxDate') && hasSubmitted"
+              >
+                {{ t("createPrices.maxDateValidationErrorText") }}
+              </watt-field-error>
+            </watt-datepicker>
```

Comment:
issue: validation errors are shown immediately after opening the page. Should be hidden until the user has interacted with the form, or submitted the form.

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/9bf87bbb-6733-4c7b-85b8-bf611e54169c)

Reply:
Should be fixed